lib/FunctionIsolation/PromoteCSVs.cpp

```diff
@@ -64,11 +66,13 @@ class PromoteCSVs {
   struct WrapperKey {
   public:
     Function *Helper;
-    std::set<GlobalVariable *> Read;
-    std::set<GlobalVariable *> Written;
+    std::vector<GlobalVariable *> ReadSortedByName;
+    std::vector<GlobalVariable *> WrittenSortedByName;
```

Comment:
I'd leave the old name and add doxygen about the fact that they are sorted.

Reply:
How about now?